test/simple/test-repl-tab-complete.js

```diff
@@ -211,3 +211,18 @@ testMe.complete(' ', function(error, data) {
 testMe.complete('toSt', function(error, data) {
   assert.deepEqual(data, [['toString'], 'toSt']);
 });
+
```

Comment:
Do these newly added tests actually test the bug that this PR fixes? It seems these tests don’t involve the debugger, so they seem not to be regression tests for this issue, or am I missing something?